hopsworks-common/src/main/resources/META-INF/beans.xml

```diff
@@ -45,7 +45,10 @@
                       http://xmlns.jcp.org/xml/ns/javaee/beans_1_1.xsd"
         bean-discovery-mode="annotated">
     <alternatives>
+        <class>io.hops.hopsworks.common.serving.LocalhostServingController</class>
         <class>io.hops.hopsworks.common.serving.tf.LocalhostTfServingController</class>
+        <class>io.hops.hopsworks.common.serving.sklearn.LocalhostSkLearnServingController</class>
```

Comment:
We don't need this one here - it's not an alternative

Reply:
Done